.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: d
+os:
+  - linux
+  - osx
```

Comment:
It already does support D on OSX, but there isn't anything platform specific in this library, so you shouldn't needlessly waste Travis' resources.